README.md

```diff
@@ -329,20 +341,68 @@ onDialNumber(data) {
 </details>
 
 <details>
- <summary>onEngagementCreated</summary>
+ <summary>onCreateEngagementSucceeded</summary>
  <p>
 
 ```js
-  // Message indicating that HubSpot has created
-  onEngagementCreated(data) {
+  // Message indicating that HubSpot has created an engagement
+  onCreateEngagementSucceeded(data) {
     const {
       /* A HubSpot created engagement id. */
       engagementId: number,
     } = data;
       ...
   }
 ```
+</p>
+</details>
+
+<details>
+ <summary>onCreateEngagementFailed</summary>
+ <p>
+
+```js
+  // Message indicating that HubSpot has failed to create an engagement
+  onCreateEngagementFailed(data) {
+    const {
+      error: Error
```

Comment:
Is there any public documentation around the /engagements API we could reference around what fields the Error object may contain?

Reply:
We're using the built-in Javascript [Error() constructor](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) currently but I think we might create an error wrapper to standardize this in the future. What do you think of using an error object for the time being?
```
const {
  error: { message: string }
}
```

Reply:
That SGTM @esme, we can get more specific with error types over time